docs/user_guide/selection/ProbeFeatureSelection.rst

```diff
@@ -0,0 +1,224 @@
+.. _probe_features:
+
+.. py:currentmodule:: feature_engine.selection
+
+ProbeFeatureSelection
+=====================
+
+ProbeFeatureSelection() generates one or more probe features based on the
```

Comment:
We need to wrap the class like these: :class:`ProbeFeatureSelection()` so that in the documentation, it links to the API part.

Reply:
I'd also add "generates one or more random features..."